src/plugins/discover/public/application/main/components/layout/discover_layout.tsx

```diff
@@ -318,7 +346,7 @@ export function DiscoverLayout({ stateContainer }: DiscoverLayoutProps) {
                 documents$={stateContainer.dataState.data$.documents$}
                 onAddField={onAddColumn}
                 columns={currentColumns}
-                onAddFilter={!isPlainRecord ? onAddFilter : undefined}
+                onAddFilter={!isPlainRecord ? onAddFilter : onPopulateWhereClause}
```

Comment:
Let's create a variable to reuse it here and above for `mainDisplay`

Reply:
https://github.com/elastic/kibana/pull/181399/commits/a6ddb094a5490fe1f2255d6cb4961e0817ace9f5